addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -744,13 +750,27 @@ var SlideUploadDialog = Dialog.extend({
             }
         }
     },
-
+    _setDefaultSelection: function () {
+        const sections = this.values.read_results
```

Comment:
check your linter

why not doing something like this ?
```js
_setDefaultSelection: function () {
    const sections = this.values.read_results;
    if (!sections.length) {
        return;
    }
    const latestSection = sections[sections.length - 1];
    this.$("#category_id").select2("data", { id: latestSection.id, text: latestSection.name });
    this.$("#category_id").select2("readonly", false);
},
```

Reply:
The controller provides data in a sequential order, but the UI allows users to rearrange sections as they like, enabling them to place the most recently created section anywhere in the sequence. So that's why I used this approach. Your proposed solution would provide the last section in the sequence, which does not align with the dynamic nature of the user interface.

Reply:
> So that's why I used this approach

You just order by id no ?

Why is it different than mine ?

If you want to order by id and not by sequence, you can also add the `order` argument when you call `search_read` in `_fetchSections`

Reply:
Done